save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -271,6 +326,9 @@ class VulnerabilityService(
                 )
             }
         )
+
+        val vulnTagLinks = existedTags.plus(newTags).map { LnkVulnerabilityTag(vulnerability, it) }
```

Comment:
`existedTags.plus(newTags)` == `vulnerabilityDto.tags`, I guess?

Reply:
`existedTags` and `newTags` are `Tag` entities, while `VulnerabilityDto.tags` are just `String`s